### ISSUE TITLE
Time zone problems with docker?  Read this

### ISSUE THREAD

**Issue by thepotoo** [niklaswall/node-red-contrib-light-scheduler#47]
tl;dr: Does your docker-compose.yml file contain?
```
volumes:
  - /etc/localtime:/etc/localtime:ro
  - /etc/timezone:/etc/timezone:ro
environment:
  TZ: America/New_York
```

Pick your TZ from [here](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones)

More info:
I suspect only TZ is needed, but it's better to have localtime and timezone set as other apps may use it.

Setting TZ in the environment is [needed as of 1.0](https://github.com/node-red/node-red-docker/blob/master/README.md).  Other issues don't mention this.

A quick check is to run date in your node-red docker's shell.  For example, if your docker instances's name is **node-red**, enter `sudo docker exec -it node-red date` which should return the right date.

Next, make sure the problem isn't with node-red: create a function node with one line:
`node.warn(new Date());`  Here's a copy/pastable test to import:
```
[{"id":"d41fd22c.b3f2f8","type":"inject","z":"4db3e574.9051ac","name":"","topic":"","payload":"","payloadType":"date","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":1160,"y":120,"wires":[["d0e6a625.128f5"]]},{"id":"d0e6a625.128f5","type":"function","z":"4db3e574.9051ac","name":"","func":"node.warn(new Date());","outputs":1,"noerr":0,"x":1330,"y":120,"wires":[[]]}]
```

Note that just injecting timestamp (which is ms since 1970) and looking at that seems to use the time on the client's machine, which may not be the problem.

Finally, make sure dusk/dawn is set to schedule only to rule out issues with incorrect lat/long.

Hope this helps somebody.

**Comment by niklaswall** [comment]
Thanks for the help-text @thepotoo, maybe I should include some more information regarding it in the documentation.

For clarity: I do not publish any docker-compose.yml or docker image file at all. If you are running this node-red plugin in a node-red docker instance it's published by someone else. If that image for any reason have the wrong timezone configured this node will not work as you expect it!
